src/bin/zmlicensectl

```diff
@@ -76,13 +76,53 @@ startLicenseDaemonService()
 	fi
 }
 
+checkNalpeironDaemonServiceRunning() {
+	if docker inspect --format '{{.State.Running}}' zimbra-nalpeiron-daemon 2>/dev/null | grep -q "false"; then
+		nalpeironDaemonServiceRunning=0
+	else
+		docker exec zimbra-nalpeiron-daemon systemctl status nalpdaemon &>/dev/null
+		if [ $? -ne 0 ]; then
+			nalpeironDaemonServiceRunning=0
+		else
+			nalpeironDaemonServiceRunning=1
+		fi
+	fi
+}
+
+startNalpeironDaemonService() {
+	err=0
+	checkNalpeironDaemonServiceRunning
```

Comment:
Right now I am not sure about how the Nalpeiron service running inside a container will behave but in future we may need to comeup with the way to restart that as well.